api/data_explorer/__main__.py

```diff
@@ -287,6 +288,33 @@ def _get_table_names():
     return table_names
 
 
+def _get_export_url_gcs_bucket():
+    """Returns export URL bucket, with no gs:// prefix.
+
+    If bucket doesn't exist, returns an empty string.
+    """
+    config_path = os.path.join(app.app.config['DATASET_CONFIG_DIR'],
+                               'deploy.json')
+    if not os.path.isfile(config_path):
+        app.app.logger.warning(
+            'deploy.json not found. Export to Saturn feature will not work. '
+            'See https://github.com/DataBiosphere/data-explorer#one-time-setup-for-export-to-saturn-feature'
+        )
+        return ''
+
+    project = _parse_json_file(config_path)['project_id']
+    bucket = project + '-export'
```

Comment:
tiny nit: maybe '-export-saturn' to make it more specific? I could potentially see someone already having a bucket named `<project>-export`

Reply:
So we recommend creating a project just for deploying data explorer (see ["We recommend creating a project"](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery/deploy), so `<project>-export` probably won't exist.

Also, Saturn is a code name. I mention "Saturn" in comments and READMEs, but when we get a real name, I'll replace "Saturn".